addons/mail/static/tests/chatter_tests.js

```diff
@@ -819,5 +821,41 @@ QUnit.test('followers widget: do not display follower duplications', function (a
     form.destroy();
 });
 
+QUnit.test('chatter: close chat window using ESCAPE key', function (assert) {
```

Comment:
test should not be named with 'chatter': it is a chat window test, so maybe, just "chatter: close chat window using ESCAPE key"

Also, this is not a chatter_test, so it should go into a new file, chat_window_test.js